Objects/typeobject.c

```diff
@@ -4797,14 +4824,12 @@ static int
 add_methods(PyTypeObject *type, PyMethodDef *meth)
 {
     PyObject *dict = type->tp_dict;
+    PyObject *name;
```

Comment:
I'd put this down where it's first used (i.e. right above line 4852 `if (isdescr) {`).